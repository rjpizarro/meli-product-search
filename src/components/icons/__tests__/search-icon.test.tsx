import React from 'react';
import renderer from 'react-test-renderer';
import SearchIcon from '../search-icon'

describe('SearchIcon', () => {
    it('should render correctly ', () => {
        const tree = renderer
            .create(<SearchIcon />)
            .toJSON()

        expect(tree).toMatchSnapshot()
    })
})

