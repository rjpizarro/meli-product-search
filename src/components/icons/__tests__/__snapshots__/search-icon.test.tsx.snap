// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchIcon should render correctly  1`] = `
<img
  alt="Search Icon"
  sizes=" (max-width: 480px) 18px, 18px, "
  src="ic_search@2x.png"
  srcSet="ic_search@2x.png 36w,
                ic_search.png 18w"
/>
`;
